tests/conftest.py

```diff
@@ -117,7 +117,7 @@ def mock_user():
 
 @pytest.fixture
 def mock_request_information():
-    request_info = RequestInformation()
+    request_info = RequestInformation(None, None)
```

Comment:
could we make those parameters optional somehow so people can use request information without having to configure the request information right away should they need to?
Plus adding required parameters is a source breaking change